core/src/test/java/com/exadel/aem/toolkit/core/injectors/models/TestModelChildInner.java

```diff
@@ -0,0 +1,25 @@
+package com.exadel.aem.toolkit.core.injectors.models;
+
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.models.annotations.DefaultInjectionStrategy;
+import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.injectorspecific.ValueMapValue;
+
+import com.exadel.aem.toolkit.api.annotations.injectors.Child;
+
+@Model(adaptables = SlingHttpServletRequest.class, defaultInjectionStrategy = DefaultInjectionStrategy.OPTIONAL)
+public class TestModelChildInner {
+    @Child
+    private Resource innerModelResource;
+
+    @ValueMapValue
```

Comment:
Can we check also usage with relative path? for example `@ValueMapValue(name="innerModelResource/title")`